edge/pkg/metamanager/process.go

```diff
@@ -549,12 +545,17 @@ func (m *metaManager) syncPodStatus() {
 			klog.Errorf("unmarshal podstatus[%s] failed, content[%s]: %v", v.Key, v.Value, err)
 			continue
 		}
-		content = append(content, podStatus)
+		if _, ok := contents[namespaceParsed]; !ok {
+			contents[namespaceParsed] = []interface{}{podStatus}
```

Comment:
Do we need this line here? Why not append directly.

Reply:
done : )
you r rigint, I thought the argument couldn't be nil.